runtime/testy/studentAPI.py

```diff
@@ -143,15 +143,12 @@ def _hibikeSubscribeDevice(self, uid, delay, params):
     self.toManager.put([HIBIKE_COMMANDS.SUBSCRIBE, [uid, delay, params]])
 
   def _hibikeGetUID(self, name):
-    try:
-      return self.sensorMappings.get(name)
-    except:
-      raise StudentAPIKeyError()
+    return self.sensorMappings.get(name)
```

Comment:
The code as before wouldn't throw an error in the first place, since we're using `.get` instead of a standard dictionary `[]` access.